.env.example

```
@@ -1,95 +1,169 @@
+QDRANT_BATCH_SIZE=256
+QDRANT_PREFETCH_SIZE=1024
 
-# Vector Service Configuration
-# Enable modular vector service implementation (default: false)
-# Set to true to use the new modular architecture for better maintainability
-USE_MODULAR_VECTORS=true
-
-# HF Text Embeddings Inference (TEI)
+# HF Text Embeddings Inference (TEI) - RTX 4070 Optimized
 TEI_URL=http://localhost:8080
 TEI_MODEL=Qwen/Qwen3-Embedding-0.6B
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The TEI_MODEL key should go before the TEI_URL key

(UnorderedKey)